### ISSUE TITLE
when saving, a glitch happens

### ISSUE THREAD

**Issue by funkyeah** [sbenhaim/protobrew#5]
after writing to an wiki page when the save button is hit and we transition to view mode, the added text first disappears then reappears. It appears its waiting for a server response and glitch length is dependent on response time.

The console show this when saving:

XHR finished loading: "http://twbrewing.com:3000/sockjs/544/074runlj/xhr_send". 
XHR finished loading: "http://twbrewing.com:3000/sockjs/544/074runlj/xhr_send". 
XHR finished loading: "http://twbrewing.com:3000/sockjs/544/074runlj/xhr". 
XHR finished loading: "http://twbrewing.com:3000/sockjs/544/074runlj/xhr".

**Comment by funkyeah** [comment]
modify the end of the saveEntry method in client.coffee to includes Entries.update({_id: entry._id}, entry)

```
Meteor.call('saveEntry', entry, reroute)
Entries.update({_id: entry._id}, entry)
Session.set("edit-mode", false)
```

**Comment by sbenhaim** [comment]
Yes, I think that's exactly right, although it seems a little hokey to have to do that explicitly.

Please commit for now, and I'll do a little research to see if there is a good way to save locally and use server methods for validated save without having to explicitly do both.

**Comment by funkyeah** [comment]
committed

**Comment by funkyeah** [comment]
a variant of this bug has come back

it looks like it has to do with the saveEntry Meteor.call 

it was modified to include the context stuff but the client side stuff I modified above wasn't also changed

**Comment by funkyeah** [comment]
not 100% reproduceable re-assigning back to me

**Comment by funkyeah** [comment]
Entry sidebar glitches well
